crates/democracy/src/lib.rs

```diff
@@ -829,6 +830,7 @@ impl<T: Config> Pallet<T> {
             for d in &depositors {
                 T::Currency::unreserve(d, deposit);
             }
+            Self::deposit_event(Event::<T>::Tabled(prop_index, deposit, depositors));
```

Comment:
Shouldn't we use a different event such that subsquare can show it was tabled by tc?